<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="charView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="elfSelect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Elf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="humanSelect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Human</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orcSelect">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Orc</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Race&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Stats&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Armor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Health&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stamina&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Magic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Strength&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knightButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Knight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wizardButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wizard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rogueButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rogue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Points Left:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="statsLeft">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;15&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="armorBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="healthBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="staminaBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="magicBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="strengthBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="classTrait">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>130</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>nameInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elfSelect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>raceSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>humanSelect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>raceSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>orcSelect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>raceSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>knightButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>classSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wizardButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>classSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rogueButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>classSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>armorBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>healthBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>staminaBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>magicBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strengthBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createChar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nameInput()</slot>
  <slot>raceSelect()</slot>
  <slot>sliderChanged()</slot>
  <slot>classSelect()</slot>
  <slot>createChar()</slot>
 </slots>
</ui>
